modules/cluster-logging-logstore-limits.adoc

```diff
@@ -39,15 +44,22 @@ spec:
         resources: <1>
           requests:
             cpu: "1"
-            memory: "16Gi"
+            memory: "64Gi"
+        proxy: <2>
+          limits:
+            memory: 100Mi
+          requests:
+             memory: 100Mi
 ----
-<1> Specify the CPU and memory requests as needed. If you leave these values blank,
+<1> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank,
+the Elasticsearch Operator sets default values that should be sufficient for most deployments. The defaults are `16Gi` for the memory request and `1m` for the CPU request.
```

Comment:
"1m"

Reply:
@mburke5678 bump on this saying the default is `1m` which is very low

Reply:
@ewolinetz "The defualt CPU request is low and should be changed as needed." ??

Reply:
no, more that `1m` is a very low value and incorrect... you adjusted it to `1`. We can remove the line "The default CPU request is low and should be changed as needed"